presubmit.py

```diff
@@ -293,21 +317,38 @@ def bool_to_returncode(success: bool) -> int:
 
 def main() -> int:
     """Check that this branch conforms to the standards of fuzzbench."""
-    logs.initialize()
     parser = argparse.ArgumentParser(
         description='Presubmit script for fuzzbench.')
     choices = [
         'format', 'lint', 'typecheck', 'licensecheck',
         'test_changed_integrations'
     ]
-    parser.add_argument('command', choices=choices, nargs='?')
+    parser.add_argument(
+        'command',
+        choices=choices,
+        nargs='?',
+        help='The presubmit check to run. Defaults to all of them')
+    parser.add_argument('--all-files',
+                        action='store_true',
+                        help='Run presubmit check(s) on all files',
+                        default=False)
+    parser.add_argument('-v', '--verbose', action='store_true', default=False)
 
     args = parser.parse_args()
+    if args.verbose:
+        logs.initialize(log_level=logging.DEBUG)
     os.chdir(_SRC_ROOT)
```

Comment:
probably move os.chdir before the logs init.

Reply:
Done.